workspace-manager-clienttests/src/main/resources/configs/BasicAuthenticated.json

```diff
@@ -1,10 +1,24 @@
 {
   "name": "BasicAuthenticated",
-  "description": "Get workspace repeatedly, concurrently. Authentication required.",
+  "description": "Concurrent tests. Authentication required.",
   "serverSpecificationFile": "workspace-dev.json",
   "kubernetes": {},
   "application": {},
   "testScripts": [
+    {
```

Comment:
Open question: how do we want to structure these test suites?

Reply:
I was thinking of having a basic suite and maybe a longer-running suite. Right now they're all pretty quick.